Client/game_sa/CRendererSA.cpp

```diff
@@ -15,6 +15,9 @@
 #include "CMatrix.h"
 #include "gamesa_renderware.h"
 
+#define SetLightColoursForPedsCarsAndObjects(fMult) ((RpLight*(__cdecl*)(float))0x735D90)(fMult)
+#define SetAmbientColours() ((RpLight*(__cdecl*)())0x735D30)()
+
```

Comment:
These methods should be moved to a specific class, but we don't have that class now.